From michael@stovenour.net Tue Jul  1 09:31:33 2003
Date: Sat, 14 Jun 2003 13:54:55 -0500
From: Michael Stovenour <michael@stovenour.net>
To: logwatch-devel@logwatch.org
Subject: [Logwatch-Devel] Added Initialization for $Detail to
    scripts/services/sendmail

    [ The following text is in the "iso-8859-1" character set. ]
    [ Your display is set for the "ISO-8859-2" character set.  ]
    [ Some characters may be displayed incorrectly. ]

$Detail was referenced in the sendmail script but was not initialized.  I
actually was interested in the 1 item that was based on Detail => 5, so I
added the initialization code to pickup this content for "-detail Med".



Index: logwatch/scripts/services/sendmail
===================================================================
RCS file: /var/cvs/logwatch/scripts/services/sendmail,v
retrieving revision 1.17
diff -u -r1.17 sendmail
--- logwatch/scripts/services/sendmail  18 Feb 2003 15:42:01 -0000      1.17
+++ logwatch/scripts/services/sendmail  14 Jun 2003 18:48:49 -0000
@@ -21,6 +21,8 @@
 my %largeHdrs;
 my %notLocal;

+$Detail = $ENV{'LOGWATCH_DETAIL_LEVEL'};
+
 while (defined($ThisLine = <STDIN>)) {
    ($QueueID) = ($ThisLine =~ m/^([a-zA-Z0-9]+): / );
    $ThisLine =~ s/^[a-zA-Z0-9]+: //;
cvs server: Diffing logwatch/scripts/shared

Michael Stovenour

_______________________________________________
Logwatch-Devel mailing list
Logwatch-Devel@logwatch.org
http://list.logwatch.org/lists/listinfo/logwatch-devel
